quinn-interop/src/client.rs

```diff
@@ -225,7 +225,7 @@ async fn h3_download_all(
 fn config(alpn: &str, suites: CipherSuite) -> Result<ClientConfig, Box<dyn std::error::Error>> {
     let mut provider = rustls::crypto::ring::default_provider();
     if let CipherSuite::Chacha20 = suites {
-        provider.cipher_suites = vec![TLS13_CHACHA20_POLY1305_SHA256];
+        provider.cipher_suites = vec![TLS13_CHACHA20_POLY1305_SHA256, TLS13_AES_128_GCM_SHA256];
```

Comment:
Nit: maybe add a comment why we need AES-128-GCM here?

Reply:
Moved it to initial-only use instead, for better conformance with the documented requirements for the test.